xrpl4j-address-codec/src/test/java/org/xrpl/xrpl4j/codec/addresses/XAddressTest.java

```diff
@@ -215,27 +214,15 @@ public void convertBetweenClassicAndXAddressTestnet(
     );
     assertThat(xAddress).isEqualTo(testnetXAddress);
 
-    assertThat(addressCodec.isValidClassicAddress(fromXAddress.classicAddress()));
-    assertThat(addressCodec.isValidXAddress(xAddress));
+    Assertions.assertThat(addressCodec.isValidClassicAddress(fromXAddress.classicAddress()));
+    Assertions.assertThat(addressCodec.isValidXAddress(xAddress));
   }
 
-  @Test
-  public void xAddressWithBadChecksum() {
-    XAddress xAddress = XAddress.of("XVLhHMPHU98es4dbozjVtdWzVrDjtV5fdx1mHp98tDMoQXa");
-
-    assertThrows(
-      EncodingFormatException.class,
-      () -> addressCodec.xAddressToClassicAddress(xAddress),
-      "Checksum does not validate"
-    );
-  }
-
-
   @Test
   public void xAddressWithBadPrefix() {
     XAddress xAddress = XAddress.of("dGzKGt8CVpWoa8aWL1k18tAdy9Won3PxynvbbpkAqp3V47g");
 
-    assertThrows(
+    org.junit.jupiter.api.Assertions.assertThrows(
```

Comment:
Plz revert this change.

```suggestion
    assertThrows(
```